packages/core/src/external/aws/s3.ts

```diff
@@ -85,4 +124,31 @@ export class S3Utils {
       Expires: durationSeconds ?? DEFAULT_SIGNED_URL_DURATION,
     });
   }
+
+  async getPresignedUploadUrl({
+    bucket,
+    key,
+    envType,
+    durationSeconds,
+  }: {
+    bucket: string;
+    key: string;
+    envType: string;
+    durationSeconds?: number;
+  }): Promise<string> {
+    const command = new PutObjectCommand({
+      Bucket: bucket,
+      Key: key,
+      ContentLength:
+        envType === "sandbox" ? UPLOAD_FILE_SIZE_LIMIT_SANDBOX : UPLOAD_FILE_SIZE_LIMIT,
```

Comment:
let's use [our constant](https://github.com/metriport/metriport/blob/d4af37283cbf289fdcbcad25c06f19238f71125d/packages/api/src/shared/config.ts#L15) - anytime we hardcode strings, this should be a red flag - unless it's a type literal

Reply:
We don't have it on core yet..